features/FEATURE_EXPERIMENTAL_API/FEATURE_PSA/TARGET_TFM/TARGET_TFM_V1_0/include/platform_multicore.h

```diff
@@ -24,6 +24,8 @@
 #define IPC_PSA_CLIENT_REPLY_NOTIFY_MASK (1 << IPC_PSA_CLIENT_REPLY_INTR_STRUCT)
 #define IPC_PSA_CLIENT_REPLY_IPC_INTR    cpuss_interrupts_ipc_8_IRQn
 
+#define IPC_PSA_MAILBOX_LOCK_CHAN        (10)
```

Comment:
where are the other channels? is this always within the limits?

Reply:
two additional channels defined above IPC_PSA_CLIENT_CALL_CHAN,  IPC_PSA_CLIENT_REPLY_CHAN. 

There are total 16 channels.